GoMain/bin/res/docker/configuration.toml

```diff
@@ -0,0 +1,70 @@
+[Writable]
```

Comment:
@suresh-lc This is also the same as @t25kim pointed out for `GoMain/bin/res/configuration.toml` that this is a sample so it is better to place this into [the samples/datastorage folder](https://github.com/lf-edge/edge-home-orchestration-go/tree/master/samples). What do you think?